src/test/ui/asm/issue-87802.rs

```diff
@@ -0,0 +1,17 @@
+// only-x86_64
```

Comment:
```suggestion
// needs-asm-support
```

Reply:
Fixed

Reply:
This causes rollup #88067 to fail because wasm does not have `reg`.

Reply:
Hmm ok, leave it as `only-x86_64` for now then.

Reply:
Reverted